README.md

```diff
@@ -442,6 +486,33 @@ proxy.close();
 
 ### Miscellaneous
 
+If you want to handle your own response after receiving the proxyRes, you can do
+so with `selfHandleResponse`
```

Comment:
A bit more clarification would be helpful here:
Not sure what it means to handle my own response. Does this mean that I will return my own response to the proxy client? Does the response have to be a modified version of the target webpage response?

A more descriptive var name would also be helpful (e.g., returnCustomResponse).

Reply:
or is selfHandleRequest is the options above the same as selfHandleResponse?

Reply:
@donasaur yea good catch it is supposed to just be selfHandleResponse in general. The feature is basically you are taking responsibility for returning whatever you want to the original response after the proxyRequest happens and returns its response. This was the name given by the contributor I'm ok with finding something that makes more sense. Its just trying to indicate that its your responsibility to return the response if thats something you needed to do. `modifyResponse` is also another idea.